service/src/main/kotlin/fi/espoo/evaka/EspooConfig.kt

```diff
@@ -34,5 +43,27 @@ class EspooConfig {
     fun templateProvider(): ITemplateProvider = EvakaTemplateProvider()
 
     @Bean
-    fun jobSchedule(env: Environment): JobSchedule = DefaultJobSchedule.fromEnvironment(env)
+    fun jobSchedule(env: ScheduledJobsEnv): JobSchedule = DefaultJobSchedule(env)
+}
+
+data class EspooEnv(val invoiceEnabled: Boolean) {
```

Comment:
I think `invoiceIntegrationEnabled` would be a better variable name since this env variable doesn't affect invoice functionality in general.